src/index.js

```diff
@@ -1,12 +1,15 @@
 export { default as Alert } from './Alert/Alert'
+export { default as Badge } from './Badge/Badge'
 export { default as Breadcrumb } from './Breadcrumb/Breadcrumb'
 export { default as Button } from './Button/Button'
 export { default as DropdownKebab } from './DropdownKebab/DropdownKebab'
+export { default as ListGroup } from './ListGroup/ListGroup'
+export { default as ListGroupItem } from './ListGroup/ListGroupItem'
 export {
   ListView,
-  ListGroupItem,
-  ListGroupItemHeader,
-  ListGroupItemContainer,
+  ListViewGroupItem,
+  ListViewGroupItemHeader,
+  ListViewGroupItemContainer,
```

Comment:
Can't it be simpler as `ListViewItem, ListViewItemHeader, ListViewItemContainer`?

Maybe it will make life easier to also export them as `ListView.Item, ListView.ItemHeader, ListView.ItemContainer` to keep them contextual.

Reply:
+1 for `ListViewItem, ListViewItemHeader, ListViewItemContainer` rename ... i'll try to add the contextual support as well. 

@jtomasek ok w/ this?

Reply:
I will go ahead and update the PR...

Reply:
@sharvit actually, this name conflicts w/ the existing `ListViewItem` (so I don't think we should change `ListViewGroupItem` to `ListViewItem`). 

If you see the ListView stories, you will see this... 

Would you still like to change contextual names in this PR? I.e. `ListView`, `ListView.Item`, `ListView.InfoItem`, `ListView.GroupItem`, `ListView.GroupItemHeader`, `ListView.Expand`, `ListView.Checkbox`, `ListView.Actions`, `ListView.MainInfo`, `ListView.Left`...

This seems to be conflating this PR a bit much i.m.o. but I can do this if we want.

Reply:
Yes, I just look deeper into the `ListView` code and it makes sense to export all of them as contextual components.
It will make the usage much simpler because you will need to import only the `ListView` to use them all.

I think it will be even better to split the file `ListView/index.js` to a single file per component.
Then the `ListView/index.js` can be something like:
```js
export { default as ListView } from './ListView';
export { default as ListViewItem } from './ListViewItem';
export { default as ListViewInfoItem } from './ListViewInfoItem';

ListView.Item = ListViewItem;
ListView.InfoItem = ListViewInfoItem;
```

Reply:
And I think that the `Icon` component can be based on a more basic `Icon` that can be reused.
We can use https://github.com/danawoodman/react-fontawesome and just export it as reusable `Icon` component.

Reply:
Sounds great! I'll make these updates soon.